internal/service/identity_api/queries.go

```diff
@@ -0,0 +1,152 @@
+// Code generated by github.com/Khan/genqlient, DO NOT EDIT.
```

Comment:
Let's add this codegen to the Makefile so others can reproduce it.

Reply:
The thing about this is that you need the GraphQL files.